x-pack/plugin/autoscaling/src/main/java/org/elasticsearch/xpack/autoscaling/action/GetAutoscalingDecisionAction.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.autoscaling.action;
+
+import org.elasticsearch.action.ActionRequestValidationException;
+import org.elasticsearch.action.ActionResponse;
+import org.elasticsearch.action.ActionType;
+import org.elasticsearch.action.support.master.AcknowledgedRequest;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.xcontent.ToXContentObject;
+import org.elasticsearch.common.xcontent.XContentBuilder;
+
+import java.io.IOException;
+
+public class GetAutoscalingDecisionAction extends ActionType<GetAutoscalingDecisionAction.Response> {
+
+    public static final GetAutoscalingDecisionAction INSTANCE = new GetAutoscalingDecisionAction();
+    public static final String NAME = "cluster:admin/xpack/autoscaling/get_autoscaling_decision";
```

Comment:
I would opt to remove the second autoscaling in the name, i.e., just:

```suggestion
    public static final String NAME = "cluster:admin/xpack/autoscaling/get_decision";
```